scripts/bootstrap.sh.in

```diff
@@ -44,6 +46,16 @@ EOF
 
 exec > >(tee -ia "${LOGFILE}") 2>&1
 
+determine_os() {
+    # This way permit to determine All the recent OS like Centos, Ubuntu
```

Comment:
I would reword as `we rely on /etc/os-release to discover the OS because its present on all recent Linux distributions`

Reply:
Better than my message indeed ! :)